NTupleMaker/bin/SynchNTupleProducer_2017.cpp

```diff
@@ -336,28 +334,7 @@ int main(int argc, char * argv[]){
 
   const bool  applyLeptonId    = cfg.get<bool>("Apply"+lep+"Id");
 
-  //dilepton veto
-  
-  float ptDiLeptonVeto     = 15.;
-  float etaDiLeptonVeto    = 2.4;
-  float dxyDiLeptonVeto    = 0.05;
-  float dzDiLeptonVeto     = 0.2;
-  bool applyDiLeptonVetoId = true;
-  bool applyDiLeptonOS     = true;
-  float isoDiLeptonVeto    = 0.3;
-  float drDiLeptonVeto     = 0.3;
-  
-  if (ch == "mt" || ch == "et") {
-    ptDiLeptonVeto     = cfg.get<float>("ptDi"+lep+"Veto");  
-    etaDiLeptonVeto    = cfg.get<float>("etaDi"+lep+"Veto");
-    dxyDiLeptonVeto    = cfg.get<float>("dxyDi"+lep+"Veto");  
-    dzDiLeptonVeto     = cfg.get<float>("dzDi"+lep+"Veto"); 
-    applyDiLeptonVetoId = cfg.get<bool>("applyDi"+lep+"VetoId");
-    applyDiLeptonOS     = cfg.get<bool>("applyDi"+lep+"OS");
-    isoDiLeptonVeto    = cfg.get<float>("isoDi"+lep+"Veto");
-    drDiLeptonVeto     = cfg.get<float>("drDi"+lep+"Veto"); 
-  }
-  
+  //Merijn removed dimuon cut information since it is in the config file, best to have in one location only to avoid confusion
 
```

Comment:
Hi Merijn, here you're removing a piece of code which allowed to read the di-muon veto quantities from the config file. Can you please explain why this is done?

Reply:
Hi Andrea,
In line  1382  We apply:  
  if(ch=="mt") otree->dilepton_veto = dilepton_veto_mt(&cfg, &analysisTree);

Thus, we directly feed the config to the function that applies the veto.

All the parameters that are read in for the dilepton veto  in the beginning of the macro are not used anywhere in the macro. 
In my opinion this does not add up to the clarity; either initialise everything here and use it later, or feed the config file, but then better don't initialise unused parameters here.